src/backlight/positions/positions.py

```diff
@@ -23,7 +23,7 @@ class Positions(pd.DataFrame):
         - `principal`: Principal in your bank account at that moment.
     """
 
-    _metadata = ["symbol"]
+    _metadata = ["symbol", "unit"]
```

Comment:
Need to add `MarketData`, `Labels`, `Trades`. maybe.

Reply:
What is the good name for `unit`?
- unit
- currency_unit
- denomination
- currency denomination

https://www.cmegroup.com/education/featured-reports/nikkei-225-spread-opportunities.html

e.g. We sometime says S&P denominated by JPY.

Reply:
In my point of view currency_unit is clear and not too long, but it's possible that I don't see some ambiguous cases. I will go for it and ctrl+f -> replace all if its not the best choice!

Reply:
I will add the parameter to the from_dataframe functions.

Reply:
Okay!